High transfer fees on testnet

Sending via the demo app on testnet is not working due to high fees requested. Demo app sends do not specify a fee token, so it's all using `AlphaUSD`.

The amount requested by the network is ~770x the amount you get from topping up at the faucet


**Error Message**

```
Caused by: TransactionExecutionError: The total cost (gas * gas fee + value) of executing this transaction exceeds the balance of the account.

This error could arise when the account does not have enough funds to:
 - pay for the total gas fee,
 - pay for the value to send.
 
The cost of the transaction is calculated as `gas * gas fee + value`, where:
 - `gas` is the amount of gas needed for transaction to execute,
 - `gas fee` is the gas fee,
 - `value` is the amount of ether to send to the recipient.
 
Request Arguments:
  from:  0xbae08717f27078aFAa377a135a970014aEd9779A
  to:    0x20c0000000000000000000000000000000000000
  data:  0xa9059cbb000000000000000000000000beefcafe54750903ac1c8909323af7beb21ea2cb0000000000000000000000000000000000000000000000000000000000b71b00

Details: insufficient funds for gas * price + value: have 1000000000000 want 774056185954260
```

**How to repro**
- go to https://internal-test.tempo.xyz/send (password is in the shared 1password)
- click connect wallet
- click fund wallet
- enter in a send
- see message in console

--- a/crates/node/src/rpc/mod.rs
+++ b/crates/node/src/rpc/mod.rs
@@ -37,7 +37,6 @@ use reth_rpc_eth_types::{
 use tempo_evm::TempoEvmConfig;
 use tempo_precompiles::contracts::provider::TIPFeeDatabaseExt;
 use tempo_primitives::TempoReceipt;
-use tempo_transaction_pool::validator::USD_DECIMAL_FACTOR;
 use tokio::sync::Mutex;
 
 /// Tempo `Eth` API implementation.
@@ -217,13 +216,7 @@ impl<N: FullNodeTypes<Types = TempoNode>> Call for TempoEthApi<N> {
         let fee_token_balance =
             self.caller_fee_token_allowance(&mut db, tx_env, evm_env.block_env.beneficiary)?;
 
-        // Fee token balance is denominated in USD Decimals and the gas allowance is expected in
-        // 10**9 so we must adjust by USD_DECIMAL_FACTOR
-        let adjusted_balance: U256 = fee_token_balance
-            .saturating_mul(USD_DECIMAL_FACTOR)
-            .saturating_to();
-
-        Ok(adjusted_balance
+        Ok(fee_token_balance
             // Calculate the amount of gas the caller can afford with the specified gas price.
             .checked_div(U256::from(tx_env.inner.gas_price))
             // This will be 0 if gas price is 0. It is fine, because we check it before.

--- a/crates/transaction-pool/src/validator.rs
+++ b/crates/transaction-pool/src/validator.rs
@@ -1,5 +1,4 @@
 use crate::transaction::TempoPooledTransaction;
-use alloy_primitives::{U256, uint};
 use reth_chainspec::{ChainSpecProvider, EthereumHardforks};
 use reth_primitives_traits::{
     Block, GotExpected, SealedBlock, transaction::error::InvalidTransactionError,
@@ -10,8 +9,6 @@ use reth_transaction_pool::{
     TransactionValidator,
 };
 use tempo_precompiles::contracts::provider::TIPFeeStateProviderExt;
-
-pub const USD_DECIMAL_FACTOR: U256 = uint!(1000_U256);
 
 /// Validator for Tempo transactions.
 #[derive(Debug)]
@@ -58,7 +55,7 @@ where
         };
 
         // Get the tx cost and adjust for fee token decimals
-        let cost = transaction.fee_token_cost().div_ceil(USD_DECIMAL_FACTOR);
+        let cost = transaction.fee_token_cost();
         if balance < cost {
             return TransactionValidationOutcome::Invalid(
                 transaction,
@@ -201,7 +198,7 @@ mod tests {
             .validate_transaction(TransactionOrigin::External, transaction.clone())
             .await;
 
-        let expected_cost = transaction.fee_token_cost().div_ceil(USD_DECIMAL_FACTOR);
+        let expected_cost = transaction.fee_token_cost();
         if let TransactionValidationOutcome::Invalid(_, err) = outcome {
             assert!(matches!(
                 err,
@@ -216,7 +213,7 @@ mod tests {
         let fee_token = token_id_to_address(1);
         let storage = vec![(
             mapping_slot(transaction.sender(), tip20::slots::BALANCES).into(),
-            transaction.fee_token_cost().div_ceil(USD_DECIMAL_FACTOR),
+            transaction.fee_token_cost(),
         )];
         let fee_token_acct = ExtendedAccount::new(0, U256::ZERO).extend_storage(storage);
         provider.add_account(fee_token, fee_token_acct);

